google/cloud/sqlalchemy_spanner/requirements.py

```diff
@@ -16,7 +16,7 @@
 from sqlalchemy.testing.requirements import SuiteRequirements
 
 
-class Requirements(SuiteRequirements):
+class Requirements(SuiteRequirements):  # pragma: no cover
```

Comment:
This class marks what features are supported by the dialect and which are not. Some of its marker-methods are used by SQLAlchemy to choose a right way to process a request, but some exist only as an information for user (and are not called by any methods). Thus, I think we can skip it in coverage.